jbpm-system-test/jbpm-tx-ejb-timers-test/.settings/.gitignore

```diff
@@ -0,0 +1,4 @@
+/org.eclipse.core.resources.prefs
+/org.eclipse.jdt.core.prefs
+/org.eclipse.m2e.core.prefs
+/org.eclipse.bpmn2.modeler.core.prefs
```

Comment:
Just Eclipse-specific files probably?

Reply:
yes, do you prefer to remove this?

Reply:
I think yes as this would mean committing IDE-specific files probably? Not sure what is the purpose of the `.settings` hidden directory.

Reply:
Removed the hidden settings directory